packages/components/src/call-out/styles.scss

```diff
@@ -0,0 +1,34 @@
+@import "@wordpress/base-styles/colors";
+@import "@wordpress/base-styles/mixins";
+@import "@wordpress/base-styles/variables";
+
+.call-out {
+	max-width: 600px;
```

Comment:
The design spec has guidance for showing 4 or 6 column wide call outs. Are these column widths encoded anywhere yet?

Should `CallOut` have a `cols` prop which sets a max width?